etc/helm/pachyderm/templates/determined/master-config.yaml

```diff
@@ -117,6 +120,25 @@ stringData:
         password: {{ required "A valid password type is required!" .Values.determined.scim.auth.password }}
         {{- end }}
     {{- end }}
+
+    {{- if .Values.determined.saml }}
```

Comment:
I am not sure if we should include this because for the unified install it can only use OIDC. Depends on what we want, keep a 1:1 mapping or only use the parts we care about?

Reply:
yea i actually agree. It was more to keep the chart in sync. its just that they are starting to diverge more and more. maybe we skip this then. lets see what @acohen4 thinks. not give the installer the option to turn something on that is not supported.